app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3 and parts[1] != parts[2]:
+        source_file = parts[1]
+        destination_file = parts[2]
+        print(1)
+        with open(source_file, "r") as file_in,\
```

Comment:
1. don't use '\' for line breaking
2. You can use `shutil.copy` to simplify all this.